### PR TITLE
Create the collection and schema automatically on genesis.

### PR DESCRIPTION
**Pull request by swatanabe**

### CODE REVIEW

File: contracts/eden/src/atomicassets.cpp [hunk 1]
```diff
@@ -51,4 +59,67 @@ namespace eden::atomicassets
...
+      else
+      {
+         actions::createcol(contract, {self, "active"_n})
+             .send(self, collection, true, std::vector{self}, std::vector{self}, 0.05,
+                   atomicassets::attribute_map{});
+      }
+
+      std::vector<format> schema{{"edenacc", "string"}, {"name", "string"},
+                                 {"img", "string"},     {"bio", "string"},
+                                 {"social", "string"},  {"inductionvid", "string"}};
```
Comment (sparkplug0025) [inline contracts/eden/src/atomicassets.cpp:92]: Can we change from `edenacc` to `account` across the board? (to support multiple communities)

@brandonfancher I think that according to the best practices meeting we need these two changes:
1. rename `inductionvid` to `video`
2. new copyrights (or attributions) field? to follow the suggestion to credit the author of the picture art?

File: contracts/eden/src/actions/genesis.cpp [hunk 2]
```diff
@@ -60,6 +61,9 @@ namespace eden
 
          induction_id++;
       }
+
+      atomicassets::init_collection(atomic_assets_account, get_self(), collection_name,
+                                    schema_name);
```
Comment (sparkplug0025) [inline contracts/eden/src/actions/genesis.cpp:66]: can we change the schema_name to `members` in our constants as well?

File: contracts/eden/src/atomicassets.cpp [hunk 3]
```diff
@@ -51,4 +59,67 @@ namespace eden::atomicassets
...
+            // This will fail if the contract is not the author of the collection.
+            // In that case, it's the responsibility of the author to make sure
+            // that notifications are set up.
+            actions::addnotifyacc(contract, {self, "active"_n}).send(collection, self);
+         }
+      }
+      else
+      {
+         actions::createcol(contract, {self, "active"_n})
+             .send(self, collection, true, std::vector{self}, std::vector{self}, 0.05,
```
Comment (sparkplug0025) [inline contracts/eden/src/atomicassets.cpp:86]: can we please move this magic 0.05 to a proper constant? (I believe it's the market fees)

File: contracts/eden/src/actions/genesis.cpp (hunk 2 above)
Comment (sparkplug0025) [reply contracts/eden/src/actions/genesis.cpp:66]: nevermind, I just remembered that the schema name is set on compile time on `EDEN_SCHEMA_NAME`

File: contracts/eden/src/atomicassets.cpp (hunk 1 above)
Comment (brandonfancher) [reply contracts/eden/src/atomicassets.cpp]: Agreed. I'll create an issue for this. Thanks!

**Review by sparkplug0025 — APPROVED** [review]

**Pull request merged**